modules/ztp-preparing-for-the-gitops-ztp-upgrade.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-preparing-for-the-gitops-ztp-upgrade_{context}"]
+= Preparing for the upgrade
+
+Use the following procedure to prepare your site for the GitOps zero touch provisioning (ZTP) upgrade.
+
+.Procedure
+
+. Obtain the latest version of the GitOps ZTP container.
+
```

Comment:
Can we add some sort of description or point to some other document that describes the ztp-site-generator container? My understanding from using it in testing is that it's just a convenient way to distribute the source CRs, example siteconfig and PGTs, and the required patches for the hub cluster.

Reply:
Added this, as per our conversation: "This allows you to easily distribute the source CRs, example siteconfig and PolicyGenTemplate files, and any required patches for the hub cluster."

Reply:
The CRs are intended to be applied to spoke clusters. Can we get some opinions from @lack on this description?

Reply:
@yliu127 Jim is out of office. What if we reword to say ""This allows you to easily distribute the source CRs, example siteconfig and PolicyGenTemplate files to the spoke clusters and any required patches for the hub cluster."

Reply:
I think the point is these CRs are meant for spoke cluster instead of hub cluster. Maybe @imiller0 can confirm when Jim is out?

Reply:
Suggested reword:
Obtain the latest version of the GitOps ZTP container from which you will extract a set of CRs used to configure the GitOps operator on the hub cluster for use in the GitOps ZTP solution.